### PR TITLE
UAF-4101 : generate use tax offset for PREQ and CM

### PR DESCRIPTION
**Pull request by lshyu**
Hi,

   Based on our groom discussion, we would like to have the same modification in KFS3. So, I located KFS3 mod and migrated to KF6 with some changes.  However, I did not remove the 'in-line' comment from KFS3.

Thanks.

### CODE REVIEW

File: kfs-purap/src/main/java/org/kuali/kfs/module/purap/document/AccountsPayableDocumentBase.java [hunk 1]
```diff
@@ -18,9 +18,13 @@
  */
 package org.kuali.kfs.module.purap.document;
 
+import java.sql.Date;
```
Comment (sskinner) [inline kfs-purap/src/main/java/org/kuali/kfs/module/purap/document/AccountsPayableDocumentBase.java]: Dead import, please remove.

File: kfs-purap/src/main/java/org/kuali/kfs/module/purap/document/AccountsPayableDocumentBase.java [hunk 2]
```diff
@@ -33,20 +37,30 @@
...
+import org.kuali.kfs.sys.service.OptionsService;
+import org.kuali.kfs.sys.service.impl.KfsParameterConstants;
 import org.kuali.kfs.vnd.businessobject.CampusParameter;
 import org.kuali.rice.core.api.util.type.KualiDecimal;
+import org.kuali.rice.coreservice.framework.parameter.ParameterService;
 import org.kuali.rice.kew.framework.postprocessor.DocumentRouteLevelChange;
 import org.kuali.rice.kim.api.identity.Person;
 import org.kuali.rice.kns.service.DataDictionaryService;
 import org.kuali.rice.krad.rules.rule.event.KualiDocumentEvent;
+import org.kuali.rice.krad.service.BusinessObjectService;
```
Comment (sskinner) [inline kfs-purap/src/main/java/org/kuali/kfs/module/purap/document/AccountsPayableDocumentBase.java]: Dead import, please remove.

**Review by sskinner — CHANGES_REQUESTED** [review]
Super small request: remove the two dead imports, and I'll get this merged directly after. Another good job!

**Comment by lshyu** [comment]
Scott,

   I removed those 2 unreferenced imports.  Please let me know if there other changes.

Thanks.

**Review by sskinner — APPROVED** [review]
Perfect, thanks for the speedy turnaround on this!

**Pull request merged**